src/atldld/cli.py

```diff
@@ -57,5 +60,76 @@ def cache_folder():
 info.add_command(version)
 info.add_command(cache_folder)
 
+
+# ============================= Dataset subcommand =============================
+
+
+@click.group(help="Commands related to atlas datasets")
+def dataset():
+    """Run dataset subcommands."""
+
+
+@click.command("info", help="Get information for a given dataset ID")
+@click.argument("dataset_id", type=int)
+def dataset_info(dataset_id):
+    """Get information for a given dataset ID."""
+    import textwrap
+
+    import atldld.dataset
+    from atldld import requests
+
+    # Send request
+    rma_parameters = requests.RMAParameters(
+        "SectionDataSet",
```

Comment:
I am kind of thinking out loud here :)

This implies that the user knows the ABI API, should we add an abstraction level here ? Something like `dataset` for `SectionDataSet` ?

Reply:
Yes, it's definitely possible. This idea also crossed my mind. I wanted to keep this class as lightweight as possible for now.

I think we can definitely think about improving it like you suggested, but I think we should also be a bit careful. In theory we could start implementing abstracting the whole content of http://api.brain-map.org/class_hierarchy (=the logic behind the interface here: http://api.brain-map.org/examples/rma_builder/rma_builder.html) just to make things complete, and it would become huge.

I can see so many things that can be generalised:
- Abstract the model names (like you suggested, corresponds to the blobs in http://api.brain-map.org/class_hierarchy)
- Abstract possible entries in the `include` parameter (they correspond to the different arrows in http://api.brain-map.org/class_hierarchy  as you click on blobs)
- Abstract possible keys in the `criteria` parameters and the possible values / value types (they correspond to the fields in the blobs as you click on them)
- More than just the `eq` operator for the criteria.
- etc.

So I guess what I'm trying to say is that their HTTP API is really complicated and making our code 100% compatible is probably impossible at the moment.

I'm super happy abstracting more things, but we should think through what we include/don't include.

Reply:
I added a follow-up issue: #48

Reply:
I totally agree with all of this! Thanks for the follow-up :)